docs/learning_neuroimaging.md

```diff
@@ -0,0 +1,46 @@
+* [In the wikibook neuroimaging data processing](https://en.wikibooks.org/wiki/Neuroimaging_Data_Processing/Data_Quality)
+* [References for quality control](https://www.zotero.org/groups/2221093/niqc)
+
+* [MRIQC](https://mriqc.readthedocs.io/en/stable/) MRI quality control. A BIDS app that runs a pipeline to assess the quality of your data.
+* [the PCP Quality Assessment Protocol](https://github.com/BIDS-Apps/QAP) is another BIDS app based on the protocol of [the connectome project data}(http://preprocessed-connectomes-project.org/quality-assessment-protocol/)
+* [Scripts for quality control of diffusion data](http://davidroalf.com/script_download/)
+* [Qoala-t](https://github.com/Qoala-T/Qoala-T) for QA for freesurfer segmentations also with an online [shinyapp](https://qoala-t.shinyapps.io/qoala-t_app/)
+* [Visual QC](https://github.com/raamana/visualqc) developed by [Pradeep](https://twitter.com/raamana_).
+
+
+# Diffusion Microstructure Imaging in Python (Dmipy)
+The Dmipy software project is dedicated to fasciliting high-level, reproducible diffusion microstructure research.
+* The [Dmipy open-source repository](https://github.com/AthenaEPI/dmipy) with many examples on implementing and fitting microstructure models.
+* Our Preliminary [Reference Paper](https://hal.inria.fr/hal-01873353/file/dmipy-diffusion-microstructure.pdf)
+
+
+# Other specific topics
+* [Computer Vision](https://neurohackademy.org/course/computer-vision/) by Michael Beyeler within Neurohackademy 2018 (53 min)
+* [Finding low-dimensional structure in large-scale neural recordings](https://neurohackademy.org/course/finding-low-dimensional-structure-in-large-scale-neural-recordings/) by Eva Dyer within Neurohackademy 2018 (1 hr 36 min)
+* [Interactive Data Visualization with D3](https://neurohackademy.org/course/interactive-data-visualization-with-d3/) by Anisha Keshavan within Neurohackweek 2017 (49 min)
+* [Neuroethics](https://neurohackademy.org/course/neuroethics/) by Eran Klein within Neurohackademy 2018 (58 min)
+* [Quantitative and diffusion MRI modeling of developmental data](https://neurohackademy.org/course/quantitative-and-diffusion-mri-modeling-of-developmental-data/) by Yason Yeatman within Neurohackweek
+
+
+# Meta analysis
+* [Overview_of_Meta-Analysis_Approaches](https://www.pathlms.com/ohbm/courses/8246/sections/12542/video_presentations/116072) by [Tom Nichols](https://twitter.com/ten_photos) at OHBM 2018 with [slides](https://figshare.com/articles/Overview_of_Meta-Analysis_Approaches/6723839) (18 min)
+* [ALE and BrainMap](https://www.pathlms.com/ohbm/courses/8246/sections/12542/video_presentations/116066) by Simon B. Eickhoff at OHBM 2018 (22 min)
+
+[NiMARE](https://github.com/neurostuff/NiMARE) is a Python library for coordinate- and image-based meta-analysis. Chris Gorgolewski wrote a [tutorial](https://www.kaggle.com/chrisfilo/coordinate-and-image-metaanalysis-with-nimare) on how to use it.
```

Comment:
Do you think  we should try applying our new "templates" to the NiMARE for example?

Reply:
Yes! 
Thoughts:
1. do you think "tutorials" should be its own category? It's probably a wider category than workshop (the closest existing template). And some "workshop" fields (date, duration) may not apply.
2. I thought about repurposing the "type" field to include multiple types of tags. Do you think this is a good idea, or would you prefer separate fields for this? I'm in favor of having fewer fields, so we don't end up with 15 fields, only some of which have entries. I've added a broad-ish neuroscience-field tag ([meta-analysis]) and the category ([tutorial]).
3. do we want to acknowledge/cite ccontent creators, e.g. here: Chris Gorgolewski ?

How about this:
```suggestion
NiMARE
 - description: NiMARE is a Python library for coordinate- and image-based meta-analysis.
 - URL: https://www.kaggle.com/chrisfilo/coordinate-and-image-metaanalysis-with-nimare
 - type: [tutorial] [juptyer][meta-analysis]
 - level: [intermediate]
```

Reply:
Just saw there is a "by" field!
```suggestion
NiMARE
 - description: NiMARE is a Python library for coordinate- and image-based meta-analysis.
 - URL: https://www.kaggle.com/chrisfilo/coordinate-and-image-metaanalysis-with-nimare
 - type: [tutorial] [juptyer] [meta-analysis]
 - level: [intermediate]
 -by: Chris Gorgolewski
```

Reply:
Ah yes but in this case NiMARE is a python package AND it has its own tutorial. So in this case the tuto is by Chris but the NiMARE has a large bunch of people in it.

I agree that tutorial should be its own entity but maybe we could nest entities within one another. 

Say we have a template for tutorials that we can use for standalone things. But if a software has a tutorial we can reuse that template within the software entity. Doest it make sense?

What do you think?

Reply:
Oh I like the approach of listing types, though I am now just tempted to just call this field litteraly "tags": what do you say?

Reply:
I was thinking exactly the same regarding naming it "tags". And nesting is a good idea! We'd have to shuffle the overall organization a bit then. e.g. Put links to tutorials under the software/package/analysis/modality they belong to (instead of having lists of tutorials directly). Might help the readability of the website even. what do you think?

Reply:
> I was thinking exactly the same regarding naming it "tags". And nesting is a good idea! We'd have to shuffle the overall organization a bit then. e.g. Put links to tutorials under the software/package/analysis/modality they belong to (instead of having lists of tutorials directly). Might help the readability of the website even. what do you think?

yes this should improve readibility. definitely. :smile: :crossed_fingers: